src/compiler/checker.ts

```diff
@@ -39276,13 +39276,34 @@ namespace ts {
             registerForUnusedIdentifiersCheck(node);
         }
 
+        function isBlockLike(node: Node): node is BlockLike {
+            switch (node.kind) {
+                case SyntaxKind.Block:
+                case SyntaxKind.SourceFile:
+                case SyntaxKind.ModuleBlock:
+                case SyntaxKind.CaseClause:
+                    return true;
+                default:
+                    return false;
+            }
+        }

```

Comment:
This should probably have been moved from `extractSymbol.ts` into `src/compiler/utils.ts` rather than being copied and pasted here.

Reply:
Yeah I was trying the tests :) Sorry for this, it's just my computer really cannot run the tests without exploding :sweat_smile: Is there any tip to run them locally without overwhelming it?